fhir-search/src/main/java/com/ibm/fhir/search/uri/UriBuilder.java

```diff
@@ -174,6 +175,15 @@ private void appendSummaryParameter() {
         }
     }
 
+    private void appendTotalParameter() {
```

Comment:
As a builder shouldn't these methods all be returning this?

Reply:
This is one of several private methods that are just used to construct an internal queryString.  They are called by the public builder method.